Src/Base/AMReX_MFIter.cpp

```diff
@@ -496,6 +496,7 @@ MFIter::operator++ () noexcept
 #endif
         
         bool use_gpu = (numOmpThreads == 1) && Gpu::inLaunchRegion();
```

Comment:
I think we should move this line into the pragma region.  `use_gpu` is used a few lines below.  That should be changed to `Gpu::inLaunchRegion()`.

Reply:
We will make gpu streams thread safe in a different PR.

Reply:
Agreed and fixed. Actually `use_gpu` didn't really express what I wanted to do (and that code won't work either way when using OpenMP) so I just made it `Gpu::inLaunchRegion()` as well.